node/sc/bridge_manager_test.go

```diff
@@ -255,15 +255,7 @@ func TestBridgeManager(t *testing.T) {
 		fmt.Println("Transfer Transaction", tx.Hash().Hex())
 		sim.Commit() // block
 
-		// TODO-Klaytn-Servicechain needs to support WaitMined
-		//timeoutContext, cancelTimeout := context.WithTimeout(context.Background(), 10*time.Second)
-		//defer cancelTimeout()
-		//
-		//receipt, err := bind.WaitMined(timeoutContext, sim, tx)
-		//if err != nil {
-		//	log.Fatal("Failed to WithdrawToken.", "err", err, "txHash", tx.Hash().String(),"status",receipt.Status)
-		//}
-		//fmt.Println("WithdrawToken is executed.", "addr", addr.String(), "txHash", tx.Hash().String())
+		CheckReceipt(sim, tx, 1*time.Second, types.ReceiptStatusSuccessful, t)
```

Comment:
Is 1 sec enough to wait for the receipt?

Reply:
Yes, it works on the simulated backend. During `sim.Commit()` the receipt can be generated. :)